platform/index.md

```diff
@@ -1,18 +1,70 @@
+
+<img src="architecture_overview.png" title="Particular Service Platform architecture">
+
+# NServiceBus - where it all begins
+
+NServiceBus is the heart of the system. It is a messaging and workflow framework that will help you to create distributed systems that are scalable, reliable and easy to modify. It supports various [messaging patterns](nservicebus/messaging/messages-events-commands.md), such as commands, events (publish-subscribe), offers support for handling long-running business processes in the form of [sagas](nservicebus/sagas/) and provides abstraction over multiple [queueing technologies](nservicebus/transports/), which guarantees that each message will be delivered once and only once. It automatically solves intermittent problems, by retrying messages and if that does not solve the issue, it sends them to the error queue, where they wait for human intervention.
+
+NServiceBus is very extensible, you can tailor it to your needs, select technologies you are already familiar with or create customized versions of various elements in the system. For more details refer to the [documentation](nservicebus/).
+
+# ServiceControl - gather all the data
+
+ServiceControl is the brain of monitoring in Particular Service Platform. It collects data on every single message flowing through your system (Audit Queue), errors (Error Queue), as well as additional information regarding sagas, endpoints heartbeats and custom checks (Control Queue). The information then is exposed to ServicePulse and ServiceInsight via HTTP API and SignalR notifications.
+
+It is important to understand, that the data is still collected even if ServiceControl is down. When it starts working again, it will process all the information that was saved in the meantime.
+
+In order to allow ServiceControl for gathering information, you need to configure your solution properly:
+
+* [enable auditing](nservicebus/operations/auditing.md) to collect data on individual messages;
+* [install plugins on your endpoints](servicecontrol/plugins.md) to monitor their health, sagas and use custom checks.
+
+By default ServiceControl stores information for 30 days and initiates purging process every minute, but you can easily customize those values using [configuration](servicecontrol/creating-config-file.md) or [command-line arguments](servicecontrol/how-purge-expired-data.md).
+
+The standard approach is to have one ServiceControl installation on a single host. ServicePulse and ServiceInsight can be installed on other machines. You need to configure ServicePulse and ServiceInsight to connect with ServiceControl. You can also make ServiceControl highly available, to avoid having a single point of failure in your monitoring system.
```

Comment:
> one ServiceControl installation on a single host

What does that mean? A single machine? host is a term we use when we talk about endpoints so is it "one SC per endpoint"?